tst/EnergyPlus/unit/EconomicTariff.unit.cc

```diff
@@ -0,0 +1,236 @@
+		"    TwoSeasonSchedule-Fmc,   !- Name                                        ",
+		"    number,                  !- Schedule Type Limits Name                   ",
+		"    Through: 5/31,           !- Field 1                                     ",
+		"    For: AllDays,            !- Field 2                                     ",
+		"    Until: 24:00,1,          !- Field 3                                     ",
+		"    Through: 9/30,           !- Field 5                                     ",
+		"    For: AllDays,            !- Field 6                                     ",
+		"    Until: 24:00,3,          !- Field 7                                     ",
+		"    Through: 12/31,          !- Field 9                                     ",
+		"    For: AllDays,            !- Field 10                                    ",
+		"    Until: 24:00,1;          !- Field 11                                    ",
+		"                                                                            ",
+		"  Schedule:Compact,                                                         ",
+		"    TimeOfDaySchedule-Fmc,   !- Name                                        ",
+		"    number,                  !- Schedule Type Limits Name                   ",
+		"    Through: 5/31,           !- Field 1                                     ",
+		"    For: AllDays,            !- Field 2                                     ",
+		"    Until: 15:00,3,          !- Field 3                                     ",
+		"    Until: 22:00,1,          !- Field 5                                     ",
+		"    Until: 24:00,3,          !- Field 7                                     ",
+		"    Through: 9/30,           !- Field 9                                     ",
+		"    For: AllDays,            !- Field 10                                    ",
+		"    Until: 10:00,3,          !- Field 11                                    ",
+		"    Until: 19:00,1,          !- Field 13                                    ",
+		"    Until: 24:00,3,          !- Field 15                                    ",
+		"    Through: 12/31,          !- Field 17                                    ",
+		"    For: AllDays,            !- Field 18                                    ",
+		"    Until: 15:00,3,          !- Field 19                                    ",
+		"    Until: 22:00,1,          !- Field 21                                    ",
+		"    Until: 24:00,3;          !- Field 23                                    ",
+		"                                                                            ",
+
+	} );
+
+	ASSERT_FALSE( process_idf( idf_objects ) );
+
+	UpdateUtilityBills();
+
+	// tariff
+	EXPECT_EQ( 1, numTariff );
+	EXPECT_EQ( "EXAMPLEFMC", tariff( 1 ).tariffName );
+	EXPECT_EQ( conversionKWH, tariff( 1 ).convChoice );
+	EXPECT_EQ( 37.75, tariff( 1 ).monthChgVal );
+
+	// qualify
+	EXPECT_EQ( 1, numQualify );
+	EXPECT_EQ( false, qualify( 1 ).isMaximum );
```

Comment:
This is probably the line that's causing the build warning.  Better to use EXPECT_FALSE( qualify . . . )  Same thing a few lines down.